sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -1,14 +1,12 @@
 import warnings
 
 import numpy as np
-import pytest
 from scipy import linalg
-
+import pytest
```

Comment:
You can revert this change. The order was fine and we don't want to have unnecessary change